### PR TITLE
Fixing bugs and removing the guessed word

### PR DESCRIPTION
**Pull request by arielcoral**

### CODE REVIEW

File: src/gameFunctionality/flipCard.ts [hunk 1]
```diff
@@ -80,8 +80,30 @@ export const flipCard = (clue: string, socket: Socket) => {
...
-        socket.emit("updateGameProperties", {
-            gameArray: gameArray
-        } as gamePropertiesObj)
-    }
+        socket.emit("updateGameProperties", {gameArray: gameArray} as gamePropertiesObj)
+        console.log("nextTurn:", nextTurn)
+        }
+        if (gamePropertiesStore.firstTeamWords?.includes(clue)){
+            const indexOfTheWord = gamePropertiesStore.firstTeamUnguessedWords?.indexOf(clue);
+            if (indexOfTheWord !== -1 && typeof indexOfTheWord === "number") {
```
Comment (OriGiladi) [inline src/gameFunctionality/flipCard.ts]: typeof indexOfTheWord === "number" 
is it necessary?

File: src/gameFunctionality/flipCard.ts [hunk 2]
```diff
@@ -80,8 +80,30 @@ export const flipCard = (clue: string, socket: Socket) => {
...
+                const updatedUnguessedWords = [...(gamePropertiesStore.firstTeamUnguessedWords || [])];
+                updatedUnguessedWords.splice(indexOfTheWord, 1);
+                console.log("index " + indexOfTheWord)
+                console.log("ariel first " + updatedUnguessedWords)
+                socket.emit("updateGameProperties", { firstTeamUnguessedWords: updatedUnguessedWords });
+            }
+        } else if (gamePropertiesStore.secondTeamWords?.includes(clue)){
+            const indexOfTheWord = gamePropertiesStore.secondTeamUnguessedWords?.indexOf(clue);
+            if (indexOfTheWord !== -1 && typeof indexOfTheWord === "number") {
+                const updatedUnguessedWords = [...(gamePropertiesStore.secondTeamUnguessedWords || [])];
```
Comment (OriGiladi) [inline src/gameFunctionality/flipCard.ts]: | | [ ] ?

File: src/gameFunctionality/flipCard.ts (hunk 1 above)
Comment (arielcoral) [reply src/gameFunctionality/flipCard.ts]: fixed

File: src/gameFunctionality/flipCard.ts (hunk 2 above)
Comment (arielcoral) [reply src/gameFunctionality/flipCard.ts]: okay, fixed.

**Pull request merged**